Doc/library/weakref.rst

```diff
@@ -95,13 +95,13 @@ See :ref:`__slots__ documentation <slots>` for details.
 
 .. class:: ref(object[, callback])
 
-   Return a weak reference to *object*.  The original object can be retrieved by
-   calling the reference object if the referent is still alive; if the referent is
-   no longer alive, calling the reference object will cause :const:`None` to be
-   returned.  If *callback* is provided and not :const:`None`, and the returned
-   weakref object is still alive, the callback will be called when the object is
-   about to be finalized; the weak reference object will be passed as the only
-   parameter to the callback; the referent will no longer be available.
+   Return a weak reference to *object*.  The referent can be retrieved by
+   calling the weak reference if the referent is still alive; if the referent is
+   no longer alive, calling the weak reference will cause :const:`None` to be
+   returned.  If *callback* is provided and not :const:`None`, and the weak
+   reference is still alive, the callback will be called when the referent is
+   about to be finalized; the weak reference will be passed as the only
+   argument to the callback; the referent will no longer be available.
```

Comment:
-1 to these changes, they do not fix problems and in some cases make text harder to understand

Reply:
I agree that these are gratuitous changes.

Reply:
Agreed as well.

Reply:
@merwok Are you specifically taking about the change of ‘original object’ to ‘referent’?

Because the other changes are part of the normalisation of the use of ‘weak reference’, instead of a mix of ‘weak reference’ (37 occurrences), ‘weak reference object’ (7 occurrences), ‘reference’ (7 occurrences), ‘reference object’ (6 occurrences), ‘weakref’ (3 occurrences), and ‘weakref object’ (1 occurrence), which I think makes the page easier to read and less ambiguous.